src/components/EditForm.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+class TodoForm extends React.Component {
+  constructor(props) {
+    super(props);
+    const todo = this.props.prevTodo;
+
+    this.state = {
+      text: todo.text
+    };
+  }
+
+  updateOnChange = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value
+    });
+  };
+
+  // checkValidInput = string => [...string].some(el => el !== ' ');
+
+  editTodo = e => {
+    e.preventDefault();
+    e.stopPropagation();
+    this.props.editTodo({
+      id: this.props.prevTodo.id,
+      text: this.state.text ? this.state.text : this.props.prevTodo.text,
```

Comment:
You should use the util function `isValidText` to here as well. Reusability :slightly_smiling_face: